setup.py

```diff
@@ -8,13 +8,13 @@
 
 setup(
     name='FoxPuppet',
-    version='0.1.0',
+    version='0.2.0',
```

Comment:
I don't think we have to bump our version number at this stage, or is there any planned release?